backend/app/summary.py

```diff
@@ -272,14 +266,17 @@ def update():
         appid = ref.split("/")[1]
 
         ini = xa_cache[ref][2]
-        parser = configparser.RawConfigParser(strict=False)
+        parser = configParserCustom.ConfigParserMultiOpt()
         parser.read_string(ini)
 
         if "Build" in parser:
             build = parser["Build"]
             if "built-extensions" in build:
-                built_ext = build["built-extensions"].split(";")
-                for ext in built_ext:
+                if isinstance(build["built-extensions"], tuple):
+                    built_ext = ";".join(build["built-extensions"]).split(";")
+                else:
+                    built_ext = build["built-extensions"].split(";")
+                for ext in filter(len, built_ext):
```

Comment:
This is a slight fix for the reverse_lookup, as we could end up with empty keys here before this